examples/variational_autoencoder.py

```diff
@@ -39,7 +41,7 @@ def sampling(args):
 
 
 def vae_loss(x, x_decoded_mean):
-    xent_loss = original_dim * objectives.binary_crossentropy(x, x_decoded_mean)
+    xent_loss = objectives.binary_crossentropy(x, x_decoded_mean)
```

Comment:
@yaringal I thought IAF was using KL in the hidden space and reparametrization trick. If that is not true than we could skip that reference.